pages/index.html

```diff
@@ -2,20 +2,24 @@
 	"title": "jQuery",
 	"pageTemplate": "page-fullwidth.php",
 	"customFields": [
-			{ "key": "hide_title", "value": 1 }
+		{ "key": "banner", "value": "jQuery 4.0 is currently in beta. Once released, jQuery 3.x will only receive critical updates. Learn more about our <a href=\"https://jquery.com/support/\">Version Support</a>." },
```

Comment:
I assume this may change if https://github.com/jquery/jquery-wp-content/pull/468 changes.

Reply:
This has been removed. See https://github.com/jquery/infrastructure-puppet/pull/64